src/twisted/web/test/_util.py

```diff
@@ -55,9 +62,9 @@ def assertFlattensImmediately(self, root, target):
             L{target}.
         @rtype: L{bytes}
         """
-        return self.successResultOf(self.assertFlattensTo(root, target))
+        return self.successResultOf(self.assertFlattensTo(root, target))  # type: ignore[no-any-return]
```

Comment:
`cast` instead

Reply:
I don't think I agree in this case. What we're working around here is not a limitation of mypy or the type system, but the fact that `successResultOf()` isn't annotated yet. I assume that will change at some point and having a `type: ignore` comment means that mypy will flag the suppression as a warning as soon as it can be removed. If we'd use a `cast`, we wouldn't be notified when the workaround becomes obsolete.

Reply:
Ah, ok

Reply:
An unnecessary `cast` will also cause complaint later, which is what I was angling for.  But I think you are right and both will.